src/mainboard/novacustom/Kconfig

```diff
@@ -0,0 +1,17 @@
+if VENDOR_NOVACUSTOM
+
+choice
+	prompt "Mainboard model"
+
+source "src/mainboard/novacustom/*/Kconfig.name"
+endchoice
+
+source "src/mainboard/novacustom/*/Kconfig"
+
+config MAINBOARD_VENDOR
+	default "Notebook"
+
+config MAINBOARD_PART_NUMBER
```

Comment:
To bym przeniósł do `src/mainboard/novacustom/ns5x/Kconfig` i otoczył `if BOARD_NOVACUSTOM_NS5X`

Reply:
Done